std/traits.d

```diff
@@ -5346,7 +5346,9 @@ convertible to a string.
  */
 template isConvertibleToString(T)
```

Comment:
It does make sense to have a way to check whether something is convertible to a string without being a string. And the fact that `isSomeString` is true for enums makes things a bit interesting. If `isConvertibleToString` were true for strings, and you didn't want strings to match, the obvious thing to do would be to combine it with `!isSomeString`, but that then has issues with enums. Assuming that we can't fix `isSomeString` so that it isn't true for enums, maybe we should have something like `isExactSomeString` (which std.conv currently has as a private helper) which then is a version of `isSomeString` that's not true for enums.